brainiak/reconstruct/iem.py

```diff
@@ -0,0 +1,290 @@
+#  Copyright 2018 David Huberdeau & Peter Kok
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""Inverted Encoding Model (IEM)
+
+    Method to decode and reconstruct features from data.
+
+    The implementation is roughly based on the following publications:
+
+    [Kok2013] "1.Kok, P., Brouwer, G. J., Gerven, M. A. J. van &
+    Lange, F. P. de. Prior Expectations Bias Sensory Representations
+    in Visual Cortex. J. Neurosci. 33, 16275–16284 (2013).
+
+    [Bouwer2009] "1.Brouwer, G. J. & Heeger, D. J.
+    Decoding and Reconstructing Color from Responses in Human Visual
+    Cortex. J. Neurosci. 29, 13992–14003 (2009).
+"""
+
+# Authors: David Huberdeau (Yale University) &
+# Peter Kok (Yale University), 2018
+
+import logging
+import numpy as np
+from sklearn import linear_model
+from sklearn.base import BaseEstimator
+import math
```

Comment:
Since the only usage of `math` is `math.pi`, I would recommend using `np.pi`, so that you don't need to import `math` at all. The following stackoverflow QA says that they are the same: https://stackoverflow.com/questions/12645547/should-i-use-scipy-pi-numpy-pi-or-math-pi

Reply:
Excellent suggestion. Changed that.